tests/Unit/WritingUtilsTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Knuckles\Scribe\Tests\Unit;
+
+use Knuckles\Scribe\Tests\BaseLaravelTest;
+use Knuckles\Scribe\Tools\WritingUtils;
+
+class WritingUtilsTest extends BaseLaravelTest
```

Comment:
No idea why I didn't have a test for this class already, so thanks for this. Now I need to add tests for the other things in that file.😅